README.md

```diff
@@ -193,6 +193,7 @@ objectives:
     value: numeric # optional, value used to compare threshold metrics. Only needed when using a thresholdMetric
     target: numeric [0.0, 1.0) # budget target for given objective of the SLO
     timeSliceTarget: numeric (0.0, 1.0] # required only when budgetingMethod is set to TimeSlices
+    timeSliceWindow: number | duration-shorthand # required only when budgetingMethod is set to TimeSlices
```

Comment:
I don't think we've got the `duration-shorthand` approach specified anywhere else yet, but I do see some `number` fields that reference something similar. 

We should probably standardize how we want to reference time durations though...

Ex. TimeWindow currently specifies `unit` and `count` separately, but AlertCondition references `1h` or `5m` examples. 

If we want to go down the shorthand route we'll probably need to supply validation/interpretation logic, but it'll clean up a bit nicer than including `unit` and `count` on everything.  

Going to discuss a bit on Slack to see how folks lean and report back!

Reply:
hey @nobl9-mikec, any update here ?
Merging this will unblock me on adding support for timeslice based budgeting in [SumoLogic-Incubator/slogen](https://github.com/SumoLogic-Incubator/slogen)

Reply:
Heyo @agaurav  - sorry about the delay! I'll take a look today

Reply:
Added a follow up for unifying duration to shorthand: https://github.com/OpenSLO/OpenSLO/issues/83